hack/lint.sh

```diff
@@ -4,5 +4,5 @@
 
 VERSION="v1.45.2"
 
-docker run --rm -v $(pwd):/app:z -w /app -e GO111MODULE=on golangci/golangci-lint:${VERSION} \
+podman run --rm -v $(pwd):/app:z -w /app -e GO111MODULE=on golangci/golangci-lint:${VERSION} \
```

Comment:
nit: perhaps add detection here if podman is present, if so, use it. Otherwise use docker